ntoskrnl/ke/amd64/cpu.c

```diff
@@ -152,31 +152,31 @@ KiGetFeatureBits(VOID)
     Prcb->InitialApicId = (UCHAR)(CpuInfo.Ebx >> 24);
 
     /* Convert all CPUID Feature bits into our format */
-    if (CpuInfo.Edx & 0x00000002) FeatureBits |= KF_V86_VIS | KF_CR4;
```

Comment:
Style nit: while touching such lines, do we want to enforce the multi-line `it () {}` pattern?

Reply:
In general I would accept, but not enforce.
In this particular case, I think the function would become unnecessarily verbose and thus less readable.

Reply:
Agreed. Readability nit: shall we vertically align `FeatureBits |= ...`, as defines are?